### DISCUSSION TITLE
When will the php 8.2 server be implemented?

### DISCUSSION THREAD

**Discussion by wesllycode** [amezmo/support#4] [Support Requests] [answered]
When will the php 8.2 server be implemented?

**Comment by rmccullagh** [comment, marked as answer]
PHP 8.2 - 8.4 are implemented on Amezmo now. 

fixed in https://github.com/amezmo/docs/pull/15 

You can now host your PHP 8.2 app on Amezmo and we have updated the [Amemzo documentation](https://www.amezmo.com/docs/php#available-php-versions) to add PHP 8.x to the supported PHP versions list.

thank you for your patience
